cluster_assign_roles.rb

```diff
@@ -420,6 +421,26 @@ def ssh(host:, username:, password:, command:, streaming: false)
     end
   end
 
+  def stop_all_clients
+    target_nodes = parse_cluster_txt.reject do |entry|
+      entry[:runlist].include?('SKIP')
+    end
+
+    target_nodes.each do |node|
+      result = ssh(host: node[:ip_address],
+                   username: 'ubuntu',
+                   password: cobbler_root_password,
+                   command: 'sudo killall -9 chef-client',
```

Comment:
Discussed with @amithkanand that the chef-client will not shutdown all the time and in practice has to be done several times.

Code in https://github.com/bloomberg/chef-bach/blob/master/repxe_host.rb#L314-L357 probably needs to be copied over so that c-a-r will have the same consistency as the repxe scripts.